docs/concepts.rst

```diff
@@ -158,6 +158,96 @@ Example DAG with functional abstraction
 
 .. _concepts:dagruns:
 
+executor_config
+===============
+
+The executor_config is an argument placed into operators that allow airflow users to override tasks
+before launch. Currently this is primarily used by the KubernetesExecutor, but will soon be available
+for other overrides.
+
+pod_override
+------------
+
+When using the KubernetesExecutor, Airflow offers the ability to override system defaults on a per-task basis.
+To utilize this functionality, create a Kubernetes V1pod object and fill in your desired overrides.
+Please note that the scheduler will override the ``metadata.name`` of the V1pod before launching it.
+
+To overwrite the base container of the pod launched by the KubernetesExecutor,
+create a V1pod with a single container, and overwrite the fields as follows:
+
+.. code-block:: python
+    volume_task = PythonOperator(
```

Comment:
Can you use an literalicnlude or exampleinclude directive to include sample code snippets? This makes the code easier to maintain as all automatic checks can verify it. Ideally this example is covered by integration/system tests to be sure it is working properly.  All guides for Google services are made this way, so you can see it in practice.